Library/SettingsNotificationCellType.swift

```diff
@@ -81,6 +81,15 @@ public enum SettingsNotificationCellType {
     }
   }
 
+  public var accessibilityElementsHidden: Bool {
+    switch self {
+    case .projectNotifications, .emailFrequency:
```

Comment:
Do we need `.emailFrequency` here at all? The cell we're using for that is `SettingsNotificationPickerCell` which doesn't seem to use this value, yet it still works.